packages/itwinui-css/src/slider/slider.scss

```diff
@@ -20,7 +19,7 @@ $tick-height: $iui-baseline; // 12px
 
   svg {
     @include iui-icon-style('m');
-    margin-top: $iui-xs;
+    margin-top: var(--iui-size-2xs);
   }
 }
```

Comment:
To solve the slider icons color problem as mentioned [here](https://github.com/iTwin/iTwinUI/pull/718#issuecomment-1213274552), I think you can include the `iui-slider-color` mixin in the `iui-icon-style` mixin or something similar.

![image](https://user-images.githubusercontent.com/45748283/184411634-299bd352-831e-4ac8-b03a-fcc5b0db35d1.png)

<details>
<summary>Expand to see old code</summary>

#### slider.scss
![image](https://user-images.githubusercontent.com/45748283/184411939-a41b930a-2f1f-4c78-9136-67dac5213a92.png)

#### mixins.scss
![image](https://user-images.githubusercontent.com/45748283/184412103-f5372f3d-e016-460c-86e9-965564c50334.png)

#### mixins.scss
![image](https://user-images.githubusercontent.com/45748283/184412201-76d2e65b-31c6-4dac-938a-1978707a60d0.png)

</details>

Reply:
That is a specifically a change we don't want to make. The `iui-icon-style` mixin already has too many side effects (hence https://github.com/iTwin/iTwinUI-icons/pull/43) considering how widely it is used.

But I think we could `@include iui-icon-color` in addition to `iui-icon-style` right here in this rule (after line 21).